debian/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+RUN apt-get update && apt-get install -qqy curl python2.7 gcc python-dev python-setuptools apt-transport-https lsb-release  && \
```

Comment:
@salrashid123 I think you shouldn't even add python2.7, google-cloud-sdk must be installing it. Same for gcc, python-devtools probably installs it. Can you try deleting everything and adding only what's directly necessary?